src/api/utils/descriptions.ts

```diff
@@ -4,10 +4,19 @@
  * SPDX-License-Identifier: AGPL-3.0-only
  */
 
+import { ApiForbiddenResponse, ApiNotFoundResponse, ApiUnauthorizedResponse, } from "@nestjs/swagger";
+
 export const unauthorizedDescription =
   'Authorization information is missing or invalid';
 export const forbiddenDescription =
   'Access to the requested resource is not permitted';
 export const notFoundDescription = 'The requested resource was not found';
 export const successfullyDeletedDescription =
   'The requested resource was sucessfully deleted';
+
+export const FullApi: MethodDecorator & ClassDecorator = function FullApi(target: any) {
```

Comment:
Our eslint config does not PascalCase, but this is of course the correct way of naming decorators.
You should add an extra line that tells eslint to ignore this one rule on this one line like we did with the MarkdownBody decorator:
https://github.com/hedgedoc/hedgedoc/blob/0c2900d1c033443c5d5d4f70dd4ebc1a3db72b51/src/api/utils/markdownbody-decorator.ts#L20-L24